lib/protobuf/tasks/compile.rake

```diff
@@ -1,13 +1,32 @@
-require 'fileutils'
+require "fileutils"
 
 namespace :protobuf do
 
-  desc 'Clean & Compile the protobuf source to ruby classes. Pass PB_NO_CLEAN=1 if you do not want to force-clean first.'
+  desc "Clean & Compile the protobuf source to ruby classes. Pass PB_NO_CLEAN=1 if you do not want to force-clean first."
   task :compile, [:package, :source, :destination, :plugin, :file_extension] do |_tasks, args|
-    args.with_defaults(:destination => 'lib')
-    args.with_defaults(:source => 'definitions')
-    args.with_defaults(:plugin => 'ruby')
-    args.with_defaults(:file_extension => '.pb.rb')
+    binpath = ::File.expand_path("../../../../bin", __FILE__)
```

Comment:
If there is a better way to do this, I'm open to suggestions.

Reply:
I think it's much better than searching local path, so :+1:

Reply:
@liveh2o How about `Gem.bin_path('protobuf', 'protoc-gen-ruby')`?